recipes/muscle/meta.yaml

```diff
@@ -1,33 +1,35 @@
 {% set name = "muscle" %}
-{% set version = "3.8.1551" %}
+{% set version = "5.0.1278" %}
```

Comment:
v5.0.1428 is out as a tagged release!
https://github.com/rcedgar/muscle/releases/tag/v5.0.1428

Reply:
Thanks @colinbrislawn, I've updated the PR accordingly.